repo_policy_compliance/policy.py

```diff
@@ -60,12 +62,25 @@ class BranchJobProperty(str, Enum):
 # Using MappingProxyType to make these immutable
 ENABLED_KEY = "enabled"
 ENABLED_RULE = MappingProxyType({ENABLED_KEY: True})
+DISABLED_RULE = MappingProxyType({ENABLED_KEY: False})
+_DEFAULT_POLICY_MAP = {
```

Comment:
Perhaps rename this to base or something?

Reply:
Updated, thank you!